src/main/java/com/scrive/pdftools/tools/flattener/FlattenerHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.scrive.pdftools.tools.flattener;
+
+import org.json.JSONObject;
+
+import java.io.ByteArrayOutputStream;
+
+public class FlattenerHandler {
+
+    public static ByteArrayOutputStream execute(JSONObject flatSpecJSON) throws Exception {
+        FlattenerSpec flatSpec = FlattenerSpec.fromJson(flatSpecJSON);
+        if (flatSpec == null) {
```

Comment:
the FlattenerSpec.fromJson() method doesn't seem able to return `null` on any control path. It seems more likely that it will throw an exception on parsing failure

Reply:
Correct. I changed it to check so that the spec have fileContent